README.md

```diff
@@ -1,15 +1,15 @@
 
-This `vmap` ad playlist is marshaled through to the `ContentFlow` SceneGraph Component which handles the stream playback. In `preprocessVmapData` we parse out this ad playlist and build simple data structures which are then referenced as part of the video position change handler (`onVideoPositionChange`) to detect when we encounter a true[X] ad pod. We then initialize and launch the true[X] ad with the `isOneStageIntegration` flag, which indicates to the TruexAdRenderer that it should defer to the host channel's video player and stream for choice card asset rendering. Note that because we defer we also need the channel to notify the TruexAdRenderer when playback has moved on past the choice card, which we do so also from `onVideoPositionChange` using the `stop` action.
+In the task, we initialize the player with video metadata and RAF with ads. A very basic use case, wrapped in `vmap-truex.xml` is the default ad payload, with a preroll and midroll, each containing a TrueX ad. The payload is meant to be a simulation of a valid ad payload from the ad server. While the preroll and midroll are also local files, the TrueX ads are real server side ads, and can be seen in `truex-pod-<preroll/midroll>`. The assumption here is the ad payload is able to be parsed by RAF via `RAF.setAdUrl(url)`. RAF is fairly flexible but it is a black box. In general as long as the ad payload is a standard format, it will parse correctly. Once RAF has been setup, playback is ready to start.
 
-The `onTruexEvent` subroutine handles true[X] events and repositions the playhead depending on whether the viewer completed an ad or not (earning an ad pod skip in the former case). 
+The core of ad handling is managed by `handleAds()`. This takes care of determining if there is a valid ad to play from `raf.getAds()`. For this sample, it supports the `<AdParameter>` tag to designate a TrueX Ad Pod. `playTrueXAd` will take care of playing a given TrueX ad pod rendered on the `adFacade` that was passed from the flow. The TAR event listener `onTruexEvent` will take care of the users interactions, which dictates if the user skips or completes the interaction, and handles if it needs to resume playback or play the other ads. It will also take care of any skip cards that are automatically handled by TAR if conditions are met. If regular ads need to be rendered for any reason (opt out, no truex ads, etc), this is handled by `m.raf.showAds()`. Special callout that RAF takes completely ownership of the thread until the ad is completed or exited that needs to be handled. Since ads are handled by pausing/hiding the content stream, there is a bit of handling to ensure that the stream resumes at the correct location after completing an ad pod.
```

Comment:
As with the rest of the README, this reads very nice! 👍 

I would reword the third sentence like so: "We check for the presence of `<AdParameters>` and `get.truex.com` in the returned URL to determine whether a true[X] ad is present."